first_hour_on_vwb/README.md

```diff
@@ -0,0 +1,16 @@
+# First Hour on Verily Workbench
+
+## Overview
+
+This workspace is designed to jumpstart new users' onboarding to Verily Workbench by demonstrating some of the core capabilities of the platform in a series of JupyterLab notebooks.
+New users who engage with this workspace will be exposed to a variety of helpful references and many examples of foundational utilities and best practices that they can then build upon for their specific use cases.
+
+## Getting started
+
+- Run the [working_with_groups.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_vwb/working_with_groups.ipynb) notebook to learn about creating and deleting [Workbench groups](https://terra-docs.api.verily.com/docs/reference/glossary/#group) and managing Workbench group membership.
+
+- Run the [working_with_data_collections.ipynb](https://github.com/DataBiosphere/terra-axon-examples/blob/main/first_hour_on_vwb/working_with_data_collections.ipynb) notebook to learn about creating Workbench [data collections](https://terra-docs.api.verily.com/docs/reference/glossary/#data-collection) and adding them to your workspaces via the Workbench [data catalog](https://terra-docs.api.verily.com/docs/reference/glossary/#data-catalog).
+
+## Feedback
+
+If you have feedback about your experience with the First Hour on Verily Workbench materials, we'd love to hear from you. Please reach out to <a href="mailto:emmarogge@google.com?subject=First Hour on VWB Feedback">Emma Rogge</a> to share your thoughts.
```

Comment:
Suggestion: create a Verily group and add yourself to it.  Then, add that group's email address to this workspace.  This will let you add other people to the support group and potentially even remove yourself, should the need arise.

Reply:
Great point -- I have now done so.